src/components/JsPsychExperiment.jsx

```diff
@@ -79,9 +83,10 @@ function JsPsychExperiment({
     };
   });
 
+  // TODO: Root is not taking up 100vh here? The <body> isn't? Are the trials causing that?
```

Comment:
Maybe the issue is higher up in the dom?

Reply:
Yeah I think it is. I'm not sure how the css is filling in the `index.js` and how that's affecting this component. I added #184 for now